### PR TITLE
Add CLI to pip 

### PR DESCRIPTION
**Pull request by angelafeliciaa**

### CODE REVIEW

File: i18nilize/src/localization.egg-info/PKG-INFO [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Metadata-Version: 2.1
```
Comment (kevinrczhang) [inline i18nilize/src/localization.egg-info/PKG-INFO:1]: probably a good idea to remove the egg-info folders

File: i18ilize.egg-info/PKG-INFO [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Metadata-Version: 2.1
```
Comment (kevinrczhang) [inline i18ilize.egg-info/PKG-INFO:1]: Remove egg-info folder

File: .github/workflows/django.yml [hunk 3]
```diff
@@ -34,4 +34,5 @@ jobs:
       run: |
         cd i18nilize
         python3 -m tests.test_read_file
-        python3 -m tests.test_parse_json
\ No newline at end of file
+        python3 -m tests.test_parse_json
```
Comment (kevinrczhang) [inline .github/workflows/django.yml:37]: love that this was included 🥂

File: i18nilize/build/lib/internationalize/helpers.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
+import json
+import sys
+
```
Comment (kevinrczhang) [inline i18nilize/build/lib/internationalize/helpers.py:3]: remove this old logic, we wont be using this anymore

File: i18nilize/build/lib/internationalize/command_line.py [hunk 5]
```diff
@@ -0,0 +1,47 @@
+#from src.internationalize.helpers import add_language
+import json
```
Comment (kevinrczhang) [inline i18nilize/build/lib/internationalize/command_line.py:2]: build/lib/internationalize seems to contain exact copies of code (including outdated ones) in other folders. Was this done intentionally?

File: i18nilize/src/internationalize/helpers.py [hunk 6]
```diff
@@ -1,7 +1,11 @@
 import json
+import sys
 import os
 import requests
 
+DEFAULT_PATH = 'src/internationalize/resources/languages.json' # to be removed after refactoring
```
Comment (kevinrczhang) [inline i18nilize/src/internationalize/helpers.py:6]: remove this, no longer needed

File: i18nilize/src/internationalize/helpers.py [hunk 7]
```diff
@@ -1,7 +1,11 @@
...
+LANGUAGES_DIR = 'src/internationalize/languages'
+
```
Comment (kevinrczhang) [inline i18nilize/src/internationalize/helpers.py:8]: add this to globals.py

File: i18nilize/src/internationalize/resources/languages.json
Comment (kevinrczhang) [inline i18nilize/src/internationalize/resources/languages.json:1]: remove the resources folder instead, looks like outdated stuff

**Review by kevinrczhang — CHANGES_REQUESTED** [review]

File: i18nilize/build/lib/internationalize/helpers.py (hunk 4 above)
Comment (steph-feng) [reply i18nilize/build/lib/internationalize/helpers.py:3]: still needed

File: i18nilize/build/lib/internationalize/command_line.py (hunk 5 above)
Comment (angelafeliciaa) [reply i18nilize/build/lib/internationalize/command_line.py]: deleted

**Review by kevinrczhang — APPROVED** [review]
:shipit:

**Pull request merged**